templates/browse.html

```diff
@@ -15,9 +17,11 @@
             Browse
           </a>
         </li>
-        <!--About Link-->
+        <!--Posts Link-->
```

Comment:
Posts leads to the same place as Browse. Should maybe be new post and lead to the new post page